exercise-2/exercise-2-6/exercise-2-6.js

```diff
@@ -0,0 +1,8 @@
+function changeContent() {
+  let rowNumber = window.prompt('Input the Row number', '1');
+  let colNumber = window.prompt('Input the Column number', '1');
+  let content = window.prompt('Input the Cell content');
+  let row =
+    document.getElementById('myTable').rows[parseInt(rowNumber) - 1].cells;
```

Comment:
@trungpham-supreme If I type a character instead of a digit. What is happening?

Reply:
@duytran-agilityio  If type a character instead of a digit. The function parseInt return NaN. The table will not change.

Reply:
Please be careful with data type when you use these parse functions. Make sure it already before using it